src/apps_plugin.c

```diff
@@ -265,6 +269,37 @@ long get_system_pid_max(void) {
 	return mpid;
 }
 
+unsigned long long get_system_hertz(void)
```

Comment:
Sorry...

On 28-04-2016 13:49, Costa Tsaousis wrote:
>
> In src/apps_plugin.c 
> <https://github.com/firehol/netdata/pull/323#discussion_r61462110>:
>
> > @@ -265,6 +269,37 @@ long get_system_pid_max(void) {
> >  	return mpid;
> >  }
> >
> > +unsigned long long get_system_hertz(void)
>
> I have moved this code to common.c, but you bring it back.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/firehol/netdata/pull/323/files/460da7b0b8e59ebcb062b3b1627b452141c1a8ca..21b2e778bcce1d92816265f64175951556e53350#r61462110>
>